staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -23,13 +23,30 @@ type CustomResourceDefinitionSpec struct {
 	// Group is the group this resource belongs in
 	Group string
 	// Version is the version this resource belongs in
+	// Should be always first item in Versions field if provided.
```

Comment:
Should equal the first item in versions.

Reply:
What does "if provided" mean?

Reply:
I understand that `Versions` is optional. That should be state in the `Versions` godoc.

Reply:
Either Version or Versions should be provided. Added to the doc.